pkgs/stdenv/generic/make-derivation.nix

```
@@ -200,17 +200,24 @@ in rec {
         // (lib.optionalAttrs (attrs ? name || (attrs ? pname && attrs ? version)) {
           name =
             let
-              staticMarker = lib.optionalString stdenv.hostPlatform.isStatic "-static";
-              name' = attrs.name or
-                "${attrs.pname}${staticMarker}-${attrs.version}";
+              # Describes the target platform of the derivation if cross compiling.
```

```suggestion
              # Indicate the host platform of the derivation if cross compiling.
```
I slightly prefer "indicate" because the config is to me more a name than description ---- there's a lot of info being left out.

---

That's a good point! Fixed.